contrib/debian/control.in

```diff
@@ -199,3 +199,9 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, make | build-essential | dpkg-dev
 Description: Template for signed fwupd package
  This package is used to control code signing by the Debian signing
  service.
+
+Package: fwupd-qubes-vm-whonix
```

Comment:
shouldn't this whole section be guarded behind something still?  I would think the package is getting created right now even if qubes is set to false.

Reply:
@superm1 I've created a separate control.qubes.in for the whonix package. It's included in the `control` if the Qubes environment flag is true.